R/dplyrVerbs.R

```diff
@@ -75,9 +95,18 @@ left_join.qseaSet <- function(x, y, by = NULL, copy = FALSE, suffix = c(".x",".y
 #' @importFrom dplyr select
 #' @param .data A qseaSet to select columns of the sampleTable from
 #' @param ... Other arguments to pass to dplyr::select
-#' @return A qseaSet object with the sampleTable changed by a call to dplyr::select
+#' @return A qseaSet object with the sampleTable changed by a call to dplyr::select. 
+#'   The sample_name and group columns will always be returned.
 #' @export select.qseaSet
 #' @export
+#' @examples
+#' # select only a few columns
+#' exampleTumourNormal %>% select(type, tumour) %>% getSampleTable()
+#' # select and rename a column
+#' exampleTumourNormal %>% select(age, renamed = tumour) %>% getSampleTable()
```

Comment:
Maybe best not to use "renamed" as the new column name? Sounds like an argument name to the function. Maybe "tumour_new = tumour" would be better?